omnibus/config/software/libpcap.rb

```diff
@@ -0,0 +1,42 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+
+name "libpcap"
+default_version "1.10.5"
+# system-probe doesn't depend on any particular version of libpcap so use the latest one (as of 2024-10-28)
+# this version should be kept in sync with the one in tasks/system_probe.py
+version "1.10.5" do
+  source sha256: "84fa89ac6d303028c1c5b754abff77224f45eca0a94eb1a34ff0aa9ceece3925"
+end
+
+license "BSD-3-Clause"
+license_file "LICENSE"
+
+relative_path "libpcap-#{version}"
+
+source url: "https://www.tcpdump.org/release/libpcap-#{version}.tar.xz"
+
+build do
+  env = with_standard_compiler_flags(with_embedded_path)
+
+  configure_options = [
+    "--disable-largefile",
```

Comment:
Is this really needed? I don't know how often we expect to load capture files larger that 4GB but is there a reason to not support it?

Reply:
Oh OK that makes sense. Thanks for detailing!